hadoop-tools/hadoop-azure/pom.xml

```diff
@@ -321,8 +321,23 @@
     <dependency>
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
+      <version>4.11.0</version>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-inline</artifactId>
+      <version>4.11.0</version>
```

Comment:
if this is new to hadoop, declare it in hadoop-project/pom.xml, with versions and exclusions, then declare here without those